content/en/docs/components/katib/trial-template.md

```diff
@@ -232,12 +235,12 @@ to know more about it.
 It is possible to use your own Kubernetes CRD or other Kubernetes resource
 (e.g. [Kubernetes `Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/))
 as a trial worker without modifying Katib controller source code and building
-the new image. As long as your CRD creates Kubernetes pods, allows to inject
+the new image. As long as your CRD creates Kubernetes Pods, allows to inject
```

Comment:
is it Pods or pods in K8s? I thought it's "pods" 😄

Reply:
I think it's called `Kubernetes Pods` :). Ref: https://kubernetes.io/docs/concepts/workloads/pods/#using-pods.